### PR TITLE
Fix `ktl iex:observer` to work on Ubuntu

### PR DESCRIPTION
**Pull request by alco**

### CODE REVIEW

File: iex-observer.sh [hunk 1]
```diff
@@ -45,7 +45,7 @@ fi
 # Trap exit so we can try to kill proxies that has stuck in background
 function cleanup {
   set +x
-  sudo sed -i '' "/${HOST_RECORD}/d" /etc/hosts
+  sudo sed -i "/${HOST_RECORD}/d" /etc/hosts
```
Comment (AndrewDryga) [inline iex-observer.sh]: I think this won't work on Mac :(

Comment (alco) [reply iex-observer.sh]: Could you please try running `sed -i'' <pattern> <file>` on macOS. Note the absence of space between `-i` and `''`. it should work on macOS 10.9+.

Comment (AndrewDryga) [reply iex-observer.sh]: @alco It only works with space:
```
$ sed -i'' "/foo/d" e.txt 
sed: 1: "e.txt": invalid command code e
```

```
$ sed -i "/fiz/d" e.txt
sed: 1: "e.txt": invalid command code e
```

Comment (alco) [reply iex-observer.sh]: Let's use this workaround then - https://stackoverflow.com/a/53408233/213682. I've pushed a new commit with the change.

**Review by AndrewDryga — APPROVED** [review]

**Pull request merged**